index.html

```diff
@@ -0,0 +1,127 @@
+
+<!-- CSS Files-->
+<link rel="stylesheet"
+	href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+<link rel="stylesheet" type="text/css"
+	href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datetimepicker/4.15.35/css/bootstrap-datetimepicker.min.css">
+<link rel="stylesheet" type="text/css" href="css/main.css">
+</head>
+
+<body>
+	<nav class="navbar navbar-inverse">
+	<div class="container-fluid">
+		<div class="navbar-header">
+			<a href="#"><img class="navbar-brand"
+				src="images/MovieFlix-logo.gif" alt="MovieFlix"></a>
+		</div>
+		<div>
+			<ul class="nav navbar-nav">
+				<li><a href="#">Browse</a></li>
+				<li><a href="#">DVD</a></li>
+				<li><a href="#"><span class="glyphicon glyphicon-search"></span>
+						Search</a></li>
+			</ul>
+			<ul class="nav navbar-nav navbar-right">
+				<li><a><span class="glyphicon glyphicon-user"></span>Welcome
+						User</a></li>
+				<li><a><input class="btn btn-danger" type="submit"
+						value="Logout"></a></li>
+			</ul>
+		</div>
+	</div>
+	</nav>
+	<div class="container">
+		<div id="carousel-example-generic" class="carousel slide">
+			<!-- Indicators -->
+			<ol class="carousel-indicators hidden-xs">
+				<li data-target="#carousel-example-generic" data-slide-to="0"
+					class="active"></li>
+				<li data-target="#carousel-example-generic" data-slide-to="1"></li>
+				<li data-target="#carousel-example-generic" data-slide-to="2"></li>
+				<li data-target="#carousel-example-generic" data-slide-to="3"></li>
+			</ol>
+
+			<!-- Wrapper for slides -->
+			<div class="carousel-inner">
+				<div class="item active">
+					<img class="img-responsive img-full" src="images/civ.jpg" alt=""
+						style="width: 1600px; height: 500px">
+					<div class="carousel-caption">
+						<h3>Upcoming Movies</h3>
+					</div>
+				</div>
+				<div class="item">
+					<img class="img-responsive img-full" src="images/elm.jpg" alt=""
+						style="width: 1600px; height: 500px">
+					<div class="carousel-caption">
+						<h3>Favorite Movies</h3>
+					</div>
+				</div>
+				<div class="item">
+					<img class="img-responsive img-full" src="images/war.jpg" alt=""
+						style="width: 1600px; height: 500px">
+					<div class="carousel-caption">
+						<h3>Upcoming Movies</h3>
+					</div>
+				</div>
+				<div class="item">
+					<img class="img-responsive img-full" src="images/sup.jpg" alt=""
+						style="width: 1600px; height: 500px">
+					<div class="carousel-caption">
+						  	<h3>Upcoming Movies</h3>
+					</div>
+				</div>
+			</div>
+
+			<!-- Controls -->
+			<a class="left carousel-control" href="#carousel-example-generic"
+				data-slide="prev"> <span class="icon-prev"></span>
+			</a> <a class="right carousel-control" href="#carousel-example-generic"
+				data-slide="next"> <span class="icon-next"></span>
+			</a>
+		</div>
+	</div>
+	<div class="container">
+		<h2>Trending Now</h2>
+		<img src="images/xx.jpg" class="img-rounded" width="224" height="236">
```

Comment:
do not use height and width attributes, use bootstrap since u have included it anyway